newrelic-integration-e2e/internal/executor/executor.go

```diff
@@ -1,22 +1,60 @@
 package executor
```

Comment:
why do we have a package executor? if the purpose is to execute the code defined in the specs, schould this part of specs?